test/unit/auth/credential.spec.ts

```diff
@@ -256,7 +257,7 @@ describe('Credential', () => {
       const c = new RefreshTokenCredential(new RefreshToken(TEST_GCLOUD_CREDENTIALS));
       return c.getAccessToken().then((token) => {
         expect(token.access_token).to.be.a('string').and.to.not.be.empty;
-        expect(token.expires_in).to.equal(ONE_HOUR_IN_SECONDS);
+        expect(token.expires_in).to.greaterThan(FIVE_MINUTES_IN_SECONDS);
```

Comment:
I noticed this test is flaky. Why did you choose > 5mins?
Another option is:
.to.be.within(ONE_HOUR_IN_SECONDS - 1, ONE_HOUR_IN_SECONDS)

Reply:
5 minutes is the skew we take into consideration when scheduling [proactive token refresh](https://github.com/firebase/firebase-admin-node/blob/hkj-build-regime/src/firebase-app.ts#L130) events. Other than that the SDK can handle any reasonable expiry value sent by the server. I'd like to treat the default 3600s as an implementation detail of the token server . WDYT?

Reply:
The 5 minute value is an arbitrary one that we have chosen. Don't you think the check should be more precise? I see the counter argument on that being a token server detail. Anyway, I don't mind either way.